### PR TITLE
Add support for mmultiscripts

### PR DESCRIPTION
**Pull request by dpvc**
This implements the output for the `mmultiscripts` MathML element.

### CODE REVIEW

File: mathjax3-ts/output/chtml/Wrappers/mmultiscripts.ts [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+    /*************************************************************/
+
+    /*
+     * @override
+     */
+    public toCHTML(parent: N) {
+        const chtml = this.standardCHTMLnode(parent);
+        const data = this.getScriptData();
+        //
+        //  Combine the bounding boxed of the pre- and post-scripts,
```
Comment (zorkow) [inline mathjax3-ts/output/chtml/Wrappers/mmultiscripts.ts]: Typo

File: mathjax3-ts/output/chtml/Wrappers/mmultiscripts.ts [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+        }
+        this.childNodes[0].toCHTML(chtml);
+        if (data.numScripts) {
+            this.addScripts(u, -v, '', data.sub, data.sup, 1, data.numScripts);
+        }
+    }
+
+    /*
+     * @param{BBox} pre   The prescript bounding box
+     * @param{BBox} post  The poscript bounding box
```
Comment (zorkow) [inline mathjax3-ts/output/chtml/Wrappers/mmultiscripts.ts]: typo

File: mathjax3-ts/output/chtml/Wrappers/mmultiscripts.ts [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+        bbox.combine(post, 0, 0);
+        return bbox;
+    }
+
+    /*
+     * Create a table with the super and subscripts properly separated and aligned.
+     *
+     * @param{number} u    The baseline offset for the superscripts
+     * @param{number} v    The baseline offset for the subscripts
+     * @param{string} pre  Either 'pre' or '' (used to create mjx-prescripts or mjx-scripts element)
```
Comment (zorkow) [inline mathjax3-ts/output/chtml/Wrappers/mmultiscripts.ts]: Why is that a string and not just a boolean?

File: mathjax3-ts/output/chtml/Wrappers/mmultiscripts.ts [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+    protected addScripts(u: number, v: number, pre: string, sub: BBox, sup: BBox, i: number, n: number) {
+        const adaptor = this.adaptor;
+        const q = (u - sup.d) + (v - sub.h);             // separation of scripts
+        const U = (u < 0 && v === 0 ? sub.h + u : u);    // vertical offset of table
+        const rowdef = (q > 0 ? {style: {height: this.em(q)}} : {});
+        const tabledef = (U ? {style: {'vertical-align': this.em(U)}} : {});
+        const supRow = this.html('mjx-row');
+        const sepRow = this.html('mjx-row', rowdef);
+        const subRow = this.html('mjx-row');
+        adaptor.append(this.chtml, this.html('mjx-' + pre + 'scripts', tabledef, [supRow, sepRow, subRow]));
```
Comment (zorkow) [inline mathjax3-ts/output/chtml/Wrappers/mmultiscripts.ts]: Just select the string here.

**Comment by dpvc** [comment]
I've addressed your comments, and am merging.

**Pull request merged**